### DISCUSSION TITLE
Question:

### DISCUSSION THREAD

**Discussion by LunarHarvey** [FunkinCrew/Funkin#1571] [Compiling help] [answered]
## What is your question?
C++ Modules for v142 build tools (x64/x86) is not appearing..What should I do?

**Comment by ghost** [comment]
Hey. You should try adding the letters one by one cause it might be experimental.

**Comment by bonkmaykr** [comment, marked as answer]
> #### Please check for duplicates or similar issues before asking your question.
> ## What is your question?
> C++ Modules for v142 build tools (x64/x86) is not appearing..What should I do?

Do you have Visual Studio Community 2019 installed?

When you installed it did you select any of the C++ related packages? Usually along the lines of "C++ tools", "C++ development on desktop", etc

**Comment by LunarHarvey** [comment]
> Hey. You should try adding the letters one by one cause it might be experimental.

Hello thank you this worked!
